editor/controllers/mtf-controller.js

```diff
@@ -81,7 +81,7 @@ angular.module('mtfApp', []).controller('mtfQuestionFormController', ['$scope',
   $scope.mtfFormData.media = [];
   $scope.editMedia = [];
   var questionInput = CKEDITOR.replace('mtfQuestion', {// eslint-disable-line no-undef
-    customConfig: CKEDITOR.basePath + "config.js",// eslint-disable-line no-undef
+    customConfig: ecEditor.resolvePluginResource('org.ekstep.questionunit', '1.0', "editor/ckeditor-config.js"),
```

Comment:
Bind the CK editor to an input field. Currently is has been harcoded in multiple places. 

From specific questionUnit plugin, call bind CkEditor method of base to bind question or option element to CK editor.